unlock-app/src/hooks/useEthPrice.ts

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from '@tanstack/react-query'
+
+export const useEthPrice = () => {
+  return useQuery({
+    queryKey: ['getEthPrice'],
+    queryFn: async () => {
+      try {
+        const response = await fetch(
```

Comment:
We already have a locksmith API for this. Let's use the same please.

Reply:
true! got it!